crates/pop-drink/src/lib.rs

```diff
@@ -6,91 +8,64 @@ use scale::Decode;
 pub use session::{error::SessionError, ContractBundle, Session, NO_SALT};
 pub use sp_io::TestExternalities;
 
+/// Error type and utilities for testing contracts using the Pop API.
 pub mod error;
 #[cfg(test)]
 mod mock;
 
+/// Types and utilities for testing smart contracts interacting with Pop Network Devnet via the pop api.
 pub mod devnet {
-	pub use pop_runtime_devnet::Runtime;
+	pub use pop_runtime_devnet::{RuntimeError::*, Runtime};
```

Comment:
Conflict exports. `Session` is exported here, while the `Session` type for testing is exported in the next line.